modules/client/examples.md

```diff
@@ -815,7 +810,7 @@ true
 
 ### Loading the list of members (TokenVoting)
 
-Retrieving all the members of a TokenVoting DAO.
+Retrieving all the members of an Token DAO.
```

Comment:
Do we want this change?

Reply:
Nope, but it was changed on the examples.md but wasnt on the example itself so when i regenereated the example it changed